integration/src/ethereum/ethereum.ts

```diff
@@ -243,6 +243,52 @@ export function ethereumTests(get: () => { wallet: core.HDWallet; info: core.HDW
       TIMEOUT
     );
 
+    test(
+      "ethSignTx() - AVAX",
+      async () => {
+        if (!wallet) return;
+        // only test keepkey AVAX signing
+        // eslint-disable-next-line jest/no-jasmine-globals, jest/no-disabled-tests
```

Comment:
comments don't seem applicable (wallet check will perform this test on any wallets with `supportsETH === true)`). we probably want a `supportsAVAX` for granularity 
```suggestion
```

Reply:
This breaks the current convention used in HDWallet where `_supportsBTC` is used to indicate `_supportsUTXO`. This isn't a good convention, so this should be addressed in a larger refactor later.